doc/classes/Camera3D.xml

```diff
@@ -55,8 +55,18 @@
 			<argument index="0" name="world_point" type="Vector3">
 			</argument>
 			<description>
-				Returns [code]true[/code] if the given position is behind the camera.
+				Returns [code]true[/code] if the given position is behind the camera (the blue part of the below diagram).
 				[b]Note:[/b] A position which returns [code]false[/code] may still be outside the camera's field of view.
+				[url=https://raw.githubusercontent.com/godotengine/godot-docs/master/img/camera3d_position_frustum.png]Camera3D is position methods[/url]
+			</description>
+		</method>
+		<method name="is_position_in_frustum" qualifiers="const">
+			<return type="bool">
+			</return>
+			<argument index="0" name="world_point" type="Vector3">
+			</argument>
+			<description>
+				Returns [code]true[/code] if the given position is inside the camera's frustum (the green part of the above diagram).
```

Comment:
I would avoid using below/above as the order of methods could change in the docs if we change their names. And we don't embed the diagram so it's even less clear, I would use in inline link instead for all methods:
```suggestion
				Returns [code]true[/code] if the given position is inside the camera's frustum. [url=https://raw.githubusercontent.com/godotengine/godot-docs/master/img/camera3d_position_frustum.png]See this diagram[/url] for an overview of position query methods.
```